ftl/core/media-check.ftl

```diff
@@ -22,6 +22,7 @@ media-check-oversize-header = Files over 100MB can not be synced with AnkiWeb.
 media-check-subfolder-header = Folders inside the media folder are not supported.
 media-check-missing-header = The following files are referenced by cards, but were not found in the media folder:
 media-check-unused-header = The following files were found in the media folder, but do not appear to be used on any cards:
+media-check-template-refs-header = Files referenced by templates directly will not be considered in use, unless their names start with an underscore. Referencing templates:
```

Comment:
```suggestion
media-check-template-references-field-header = Anki can not detect used files when you use { "{{Field}}" } references in media/LaTeX tags. The media/LaTeX tags should be placed on individual notes instead.

Referencing templates:
```

Reply:
I just noticed the changed text doesn't mention static tags like `<img src=logo.jpg>` anymore. I assume this is in anticipation of the mentioned change to Check Media below? Should I then first update Check Media and later return to this PR to adjust it to the new reality?

Reply:
I don't have a preference on the order we tackle these - just think it's probably not worth mentioning static references if we'll handle them better in the future anyway.